http-body/src/lib.rs

```diff
@@ -44,6 +44,21 @@ pub trait Body {
 
     #[allow(clippy::type_complexity)]
     /// Attempt to pull out the next data buffer of this stream.
```

Comment:
I know this isn't part of the PR, but I just noticed this is old. I think this is when the functions were separate, poll_data and poll_trailers. It should mention just the "next frame" instead of data specifically.

If you want to fix it up :)

Reply:
yes! i didn't want to change it unprompted, but i can do that.

eeafd33